Makefile

```diff
@@ -925,3 +924,100 @@ ui-development: build/api build/api-db build/local-api-data-watcher-pusher build
+	./local-dev/helm repo add bitnami https://charts.bitnami.com/bitnami
+
+.PHONY: kind/cluster
+kind/cluster: local-dev/kind
+	# these IPs are a result of the docker config on our build nodes
+	export KUBECONFIG=$$(mktemp) \
+		KINDCONFIG=$$(mktemp ./kindconfig.XXX) \
+		&& chmod 644 $$KUBECONFIG \
+		&& curl -sSLo $$KINDCONFIG https://raw.githubusercontent.com/uselagoon/lagoon-charts/$(CHARTS_TREEISH)/test-suite.kind-config.yaml \
+		&& echo '  [plugins."io.containerd.grpc.v1.cri".registry.configs."registry.192.168.48.2.nip.io:32443".tls]'  >> $$KINDCONFIG \
+		&& echo '    insecure_skip_verify = true'                                                                    >> $$KINDCONFIG \
+		&& echo '  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.192.168.48.2.nip.io:32080"]'      >> $$KINDCONFIG \
+		&& echo '    endpoint = ["http://registry.192.168.48.2.nip.io:32080"]'                                       >> $$KINDCONFIG \
+		&& echo '  [plugins."io.containerd.grpc.v1.cri".registry.configs."registry.192.168.160.2.nip.io:32443".tls]' >> $$KINDCONFIG \
+		&& echo '    insecure_skip_verify = true'                                                                    >> $$KINDCONFIG \
+		&& echo '  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.192.168.160.2.nip.io:32080"]'     >> $$KINDCONFIG \
+		&& echo '    endpoint = ["http://registry.192.168.160.2.nip.io:32080"]'                                      >> $$KINDCONFIG \
+		&& echo '  [plugins."io.containerd.grpc.v1.cri".registry.configs."registry.172.25.0.2.nip.io:32443".tls]'    >> $$KINDCONFIG \
+		&& echo '    insecure_skip_verify = true'                                                                    >> $$KINDCONFIG \
+		&& echo '  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.172.25.0.2.nip.io:32080"]'        >> $$KINDCONFIG \
+		&& echo '    endpoint = ["http://registry.172.25.0.2.nip.io:32080"]'                                         >> $$KINDCONFIG \
+		&& echo 'nodes:'                                                                                             >> $$KINDCONFIG \
+		&& echo '- role: control-plane'                                                                              >> $$KINDCONFIG \
+		&& echo '  image: $(KIND_IMAGE)'                                                                             >> $$KINDCONFIG \
+		&& echo '  extraMounts:'                                                                                     >> $$KINDCONFIG \
+		&& echo '  - containerPath: /var/lib/kubelet/config.json'                                                    >> $$KINDCONFIG \
+		&& echo '    hostPath: $(HOME)/.docker/config.json'                                                          >> $$KINDCONFIG \
+		&& KIND_CLUSTER_NAME="$(CI_BUILD_TAG)" ./local-dev/kind create cluster --config=$$KINDCONFIG \
+		&& echo -e 'Interact with the cluster during the test run like so:\n' \
```

Comment:
this is meant to connect from a local host to the running jenkins in order to interact during testing? As the generated command doesn't work locally, maybe we can make this clearer?

Reply:
Sure, I'll amend the instructions.

Reply:
I added some more instructions here for local development.